projects/swimlane/ngx-ui/src/lib/components/input/input-autosize.directive.spec.ts

```diff
@@ -28,10 +30,63 @@ describe('AutosizeDirective', () => {
     expect(component).toBeDefined();
   });
 
-  it('should adjust size on input', () => {
-    const spy = spyOn(component.autosize, 'adjust');
-    component.value = 'ttttttttttttttttttttttttttttttttttt';
-    component.autosize.onInput();
-    expect(spy).toHaveBeenCalled();
+  it('should be enabled', () => {
+    expect(component.autosize?.enabled).toBeTruthy();
+  });
+
+  describe('input', () => {
+    beforeEach(() => {
+      component.type$.next(InputTypes.text);
+      fixture.detectChanges();
+    });
+
+    it('should be defined', () => {
+      expect(component.input.nativeElement).toBeTruthy();
+    });
+
+    it('should set width on input', () => {
```

Comment:
can add some tests related to honoring min/max width constraints

Reply:
good call ill add some